include/libp2p/multi/content_identifier_codec.hpp

```diff
@@ -40,6 +40,9 @@ namespace libp2p::multi {
     /// Encodes arbitrary byte buffer into CID v.0 wire format
     static std::vector<uint8_t> encodeCIDV0(const void *byte_buffer, size_t sz);
 
+    /// Encodes arbitrary byte buffer into CID v.1 wire format
+    static std::vector<uint8_t> encodeCIDV1(MulticodecType::Code content_type, const Multihash& mhash);
```

Comment:
Non-critical: a couple of files could be clang-formatted
```
include/libp2p/multi/content_identifier_codec.hpp:44
include/libp2p/protocol/kademlia/common.hpp
```

Reply:
Yep